airflow-core/tests/unit/core/test_core.py

```diff
@@ -24,10 +24,10 @@
 import pytest
 
 from airflow._shared.timezones.timezone import datetime
-from airflow.exceptions import AirflowTaskTimeout
 from airflow.models.baseoperator import BaseOperator
 from airflow.providers.standard.operators.empty import EmptyOperator
 from airflow.providers.standard.operators.python import PythonOperator
+from airflow.sdk.exceptions import AirflowTaskTimeout
```

Comment:
Why does a test in core depend on an exception from the SDK? Should the test that uses it be moved to sdk?